website/static/_redirects

```diff
@@ -609,6 +608,8 @@ https://tutorial.getdbt.com/* https://docs.getdbt.com/:splat 301!
 /date-trunc-sql /blog/date-trunc-sql 301
 /docs/using-hooks / 301
 
+/data-testing-why-you-need-it-and-how-to-get-started /blog/data-quality-testing/ 301
```

Comment:
This one is 404ing because it needs to be an absolute URL (since it's a link to www)